### ISSUE TITLE
Ledger fails to connect. Could not open client or get fingerprint information: open failed", "code": -13.

### ISSUE THREAD

**Issue by verlib** [sparrowwallet/sparrow#654]
I installed udev rules as specified in the link bellow, but my ledger still fails to connect. 
https://github.com/bitcoin-core/HWI/tree/master/hwilib/udev
when I enumerate i get the next error:
[{"type": "ledger", "model": "ledger_nano_s", "label": null, "path": "1-4:1.0", "error": "Could not open client or get fingerprint information: open failed", "code": -13}]

when i run  sudo /tmp/hwi-2.1.114749377835016321349.tmp installudevrule, i get the following error:
{"error": "installudevrules failed: [Errno 6] No such device or address", "code": -13}
![immagine](https://user-images.githubusercontent.com/11599534/184453279-8370ccd5-1043-48b5-8e85-e98340d28a97.png)

how can I install the udev rules in order to make sure my ledger finally connects?  
Ledger nano S fw 2.1.0 Sparrow 1.6.6 on Debian 11
Thanks in advance

**Comment by craigraw** [comment]
There are a few things to look at:

1. Make sure Ledger Live is not running (reboot to be sure)
2. Ensure the Bitcoin app is open on the device
3. Ensure udev rules are installed (if you followed the directions in the link you posted, that should be done)

**Comment by verlib** [comment]
1. I did make sure. I have not even installed Ledger Live on my OS. 
2. I did make sure the Bitcoin app is always open. 
3. I followed the directions just by copy-pasting the commands, but how can I check if the udev rules are correctly installed? I suppose here lies the problem. It seems the 20-hw1.rules (Ledger) are copied in the right directory, but still doesn't work
![immagine](https://user-images.githubusercontent.com/11599534/185678877-c0bbec8d-a03c-4b41-934d-f800b4996774.png)

Also, when i run sudo /tmp/hwi-2.1.114749377835016321349.tmp installudevrule, as recommended from Sparrow wallet, i get the following error:
{"error": "installudevrules failed: [Errno 6] No such device or address", "code": -13}. Why is that?
